### DISCUSSION TITLE
LDMSCON2021 -- OCT 26-28  (virtual)!

### DISCUSSION THREAD

**Discussion by oceandlr** [ovis-hpc/ldms#794] [Meetings]
LDMSCON2021 Dates Announced! OCT 26-28
Conference wiki https://sites.google.com/view/ldmscon2021/

Call for user talks coming soon!
Post tutorial topic suggestions here!

**Comment by valleydlr** [comment]
I would like to see a tutorial on a quick-start monitoring package with hands-on installation and data browsing as part of the tutorial.

**Comment by valleydlr** [comment]
Advanced features from last year needs to be lengthened in time

**Comment by vsurjadidjaja** [comment]
Breaking out Streams with Kokkos

**Reply by valleydlr** [reply]
How about inclusion of Darshan and Caliper data along with data browsing?

**Reply by valleydlr** [reply]
Preferably hands on using VMs or some such

**Comment by oceandlr** [comment]
We could drop "Writing Samplers and Stores", even though it is well attended, since it won't have changed much. We can concentrate on the new material.

**Comment by morrone** [comment]
We're probably done discussing LDMSCON2021. :)
